--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,7 +34,7 @@
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(SOFTWARE_VER "8.2.0")
-set(MXL_RV_VER "2.5")
+set(MXL_RV_VER "2.6")
 
 INCLUDE (CheckIncludeFiles)
 INCLUDE (CheckFunctionExists)
@@ -78,7 +78,7 @@ OPTION(DISABLE_INT_TIMER "Disable interv
 OPTION(HAVE_SENDMMSG "Enable/Disable use of SendMMsg()" ON)
 OPTION(HAVE_RECVMMSG "Enable/Disable use of RecvMMsg()" ON)
 OPTION(HAVE_GSO "Enable/Disable use of Generic Segmentation Offload (GSO)" ON)
-OPTION(RATE_LIMITING "Enable/Disable rate limiting via bandwidth management" OFF)
+OPTION(RATE_LIMITING "Enable/Disable rate limiting via bandwidth management" ON)
 OPTION(AUTH_IS_OPTIONAL "Make authentication optional (considered low security and should be temporary)" OFF)
 OPTION(SUPP_INVPDU_ALERT "Suppress alert when invalid control PDU is received (silently ignore)" OFF)
 OPTION(SUPP_INVPDU_WARN "Suppress warning when invalid data PDU is received (silently ignore)" OFF)
